src/olympia/files/utils.py

```diff
@@ -140,6 +146,13 @@ class InvalidZipFile(forms.ValidationError):
     pass
 
 
+class AlreadyUsedUpload(forms.ValidationError):
+    """Error raised when trying to use a FileUpload that has already been used
+    to make a Version+File, its path will be empty."""
+
+    pass
```

Comment:
I see this is a common pattern in our codebase, but why don't we specify an error string in the class directly for a specific error like this, rather than have to do it where it's used?

Reply:
`ValidationError` requires a message to be passed when instantiated anyway, so we'd have to make something really custom to avoid that... We could, bu :shrug:

Reply:
something something `__init__` something something kwarg defaults but 🤷 too - just thinking out loud at us creating specific subclasses of `ValidationError` for exceptions but then not adding anything which would render differently than a vanilla `ValidationError`